internal/data-ingest-cli/main.go

```diff
@@ -16,8 +17,8 @@ func main() {
 	collectorURL := flag.String("collector-url", "localhost:4317", "URL of the OpenTelemetry collector")
 	outputFile := flag.String("output-file", "", "Path to the file where received OTLP data will be stored")
 	inputFormat := flag.String("input-format", "otlp-json", "Input format (options: 'otlp-json', 'syslog', 'statsd')")
-	statsdProtocol := flag.String("statsd-protocol", "udp4", "Statsd protocol to send metrics (options: 'udp', 'udp4', 'udp6', 'tcp', 'tcp4', 'tcp6', 'unixgram')")
```

Comment:
why did you remove the statsd?

Reply:
Whoops. My editor must have done that, I'm not sure what happened. I undid all of these changes.